### DISCUSSION TITLE
[Toast] Accessibility considerations of stacking toasts?

### DISCUSSION THREAD

**Discussion by kylemh** [radix-ui/primitives#1637] [Help] [answered]
I see that the API allows me to render many toasts in one toast provider: https://www.radix-ui.com/docs/primitives/components/toast#duplicate-toasts

That being said, I'm not certain if I'm fucking things up in terms of accessibility! It looks like the toast items are [rendered as a list](https://github.com/radix-ui/primitives/blob/main/packages/react/toast/src/Toast.tsx#L549) and the [provider is the live region](https://github.com/radix-ui/primitives/blob/main/packages/react/toast/src/Toast.tsx#L533). So, I think I'm in the clear.

If I render toasts as a stack (with most recent in the foreground), I can use `flex-direction: column-reverse` to maintain the ideal visual and focus order, but I don't know if I could be missing anything else...

**Comment by andy-hook** [comment, marked as answer]
@kylemh Sounds like you're good, you're correct to be using layout styles to order the toasts, we manage the tab order and keyboard a11y internally based on recency rather than browser behaviour due to how portals work with DOM so providing you stick to styles for the visual "stacking" you should be fine 👍
